### PR TITLE
LPD-28718 Fix Playwright tests

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

7 files changed, 252 insertions(+), 71 deletions(-)

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:playwright

**Comment by welitonpe** [comment]
ci:relevant

**Comment by welitonpe** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:test:playwright

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by welitonpe** [comment]
ci:test:playwright

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/marketplace-publisher-dashboard/marketplaceAppPublisher.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+import {getRandomInt} from '../../../../utils/getRandomInt';
+import {marketplacePagesTest} from '../fixtures/marketplacePages';
+import {marketplaceSiteFixture} from '../fixtures/marketplaceSite';
+import {PublishProductPayload} from '../types';
+import {products} from '../utils/constants';
+
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
```
Comment (kevenleone) [inline modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/marketplace-publisher-dashboard/marketplaceAppPublisher.spec.ts:13]: @welitonpe this license should be displayed on the top 

can you fix please? after that you can forward the pr

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:playwright

**Review by joanagrego — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**